timezone_converter/comparison_view.py

```diff
@@ -70,6 +77,6 @@ def _build_table(self) -> Table:
 
         return table
 
-    def print_table(self) -> int:
-        self._print_with_rich(self._build_table())
+    def print_table(self, single_hour) -> int:
+        self._print_with_rich(self._build_table(single_hour))
```

Comment:
As suggested on `main.py`, we would use it as instance argument, so no need to pass it here.